auditlog/models.py

```diff
@@ -480,12 +478,29 @@ def changes_display_dict(self):
                         value = f"{value[:140]}..."
 
                     values_display.append(value)
-            verbose_name = model_fields["mapping_fields"].get(
-                field.name, getattr(field, "verbose_name", field.name)
-            )
+            verbose_name = self.verbose_name(field_name)
             changes_display_dict[verbose_name] = values_display
         return changes_display_dict
 
+    def verbose_name(self, field_name: str):
+        from auditlog.registry import auditlog
+
+        model = self.content_type.model_class()
+        model_fields = auditlog.get_model_fields(model._meta.model)
+        mapping_field_name = model_fields["mapping_fields"].get(field_name)
+        if mapping_field_name:
+            return mapping_field_name
+        else:
+            try:
+                field = model._meta.get_field(field_name)
+                verbose_name = getattr(field, "verbose_name", None)
+                return (
+                    verbose_name.lstrip().capitalize() if verbose_name else field_name
```

Comment:
I think we can remove the `capitalize` here to be backward compatible

Reply:
the idea here was to display field name similar to django-admin's "pretty_name" function.
thinking about it, it might be better to actually use that function rather than adding more logic.